api/nakadi-event-bus-api.yaml

```diff
@@ -799,6 +799,103 @@ paths:
             $ref: '#/definitions/Problem'
 
 
+  /subscriptions/{subscription_id}/offsets:
```

Comment:
yes, this is what I had in the beginning :)
I just thought that for user it will be more clear what 'offsets' mean.
But in general 'cursors' will be more consistent, as we really mean cursors (offsets per partition)
So I will change that to 'cursors'